section11/numberformat_proposed.html

```diff
@@ -8,28 +8,38 @@ <h1>Abstract Operations For NumberFormat Objects</h1>
       <h1>SetNumberFormatDigitOptions ( _intlObj_, _options_, _mnfdDefault_, _mxfdDefault_ )</h1>
 
       <p>
-        The abstract operation SetNumberFormatDigitOptions applies digit
-        options used for number formatting onto the intl object.
+        The abstract operation SetNumberFormatDigitOptions resolves the user-specified options in the digits realm onto the digits options object.
```

Comment:
What do you mean by the term "realm" here? That's a specialized word in TC39, which I don't understand the use of here.

Reply:
I'm not sure where that word came from here.  This description does not need to be changed from the original.